verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -323,16 +374,21 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
         this.unlisten();
     }
 
-    deleteCase(rowData: TableRow): Promise<unknown> {
-        return new Promise((resolve, reject) => {
-            const deleteUrl = this.state.url + rowData.id;
-            this.setState({ error: '' });
-            const response = axios.delete(deleteUrl);
-            response.then(resolve).catch((e) => {
-                this.setState({ error: e.toString() });
-                reject(e);
-            });
+    async deleteCases(): Promise<void> {
+        if (this.hasSelectedRowsAcrossPages()) {
+            // TODO: Implement action for all rows
+            alert('Action not yet implemented when all rows selected');
+            return;
+        }
+        await axios.delete('/api/cases', {
```

Comment:
do we want a progress bar somewhere? this could take some time.

Reply:
There is a slight lag, but the good news is it doesn't seem to be any longer if selecting more rows (I tried with 100 and it wasn't noticeably longer than 1 case). I tried quickly to put a loading spinner instead of the yes button, but it adds even more lag since it changes the components state causing the entire thing to re-render. We could potentially pull this dialog out into it's own component (and potentially share with the RowMenu dialog?) but that would be quite a big refactor. I don't think the lag is great enough to necessitate that yet, but we can see how it is on dev and make that change if we'd like to.

Reply:
ok fine for now, we will definitely need this for bigger queries I don't expect a slowdown from 1 to 100 queries but when we start hitting the thousands/hundreds of thousands yes it will definitely be needed.